src/lxc/lxc_init.c

```diff
@@ -92,6 +92,86 @@ static struct arguments my_args = {
 	.shortopts = short_options
 };
 
+static void prevent_forking(void)
+{
+	FILE *f;
+	char name[PATH_MAX], path[PATH_MAX];
```

Comment:
We actually use an internal macro `MAXPATHLEN` all across the codebase. Would be good if you could use this here too. :)